From 96c02e485df801353721b3976f6d8b5ed30bfbf3 Mon Sep 17 00:00:00 2001
From: Howard Hsu <howard-yh.hsu@mediatek.com>
Date: Thu, 5 Dec 2024 17:38:02 +0800
Subject: [PATCH 61/95] mtk: mt76: mt7996: enable qos program related bit in
 extended capabilities

Enable QoS Map information bit and mirrored scs bit in extended
capabilities for ap mode.

Signed-off-by: Howard Hsu <howard-yh.hsu@mediatek.com>
---
 mt7996/init.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mt7996/init.c b/mt7996/init.c
index e1f741d8..3c2a7ae6 100644
--- a/mt7996/init.c
+++ b/mt7996/init.c
@@ -81,8 +81,10 @@ static const struct ieee80211_iface_combination if_comb_7992 = {
 static const u8 mt7996_if_types_ext_capa_ap[] = {
 	[0] = WLAN_EXT_CAPA1_EXT_CHANNEL_SWITCHING,
 	[2] = WLAN_EXT_CAPA3_MULTI_BSSID_SUPPORT,
+	[4] = WLAN_EXT_CAPA5_QOS_MAP,
 	[6] = WLAN_EXT_CAPA7_SCS_SUPPORT,
 	[7] = WLAN_EXT_CAPA8_OPMODE_NOTIF,
+	[10] = WLAN_EXT_CAPA11_MIRRORED_SCS_SUPPORT,
 };
 
 static const u8 mt7996_if_types_ext_capa_sta[] = {
-- 
2.45.2

